src/sentry/incidents/models.py

```diff
@@ -704,6 +704,11 @@ class Meta:
         unique_together = (("alert_rule_trigger", "query_subscription"),)
 
 
+class AlertRuleTriggerActionStatus(Enum):
+    ACTIVE = 0
+    SCHEDULED_FOR_DELETION = 1
```

Comment:
Should we just use https://github.com/getsentry/sentry/blob/4b09a7178f36d17b13d69fff80a254a45b9116c1/src/sentry/constants.py#L471-L475 here?

Reply:
ohhh, good idea, let me update

Reply:
Can't tell if we should use pending_deletion or deletion_in_progress when we schedule it though. They sound like they both work? haha

Reply:
Generally when deletes are scheduled we use `pending_deletion`. Once the deletion has started, the status goes to `deletion_in_progress`. Only very large objects (like organizations) spend a measurable amount of time in `deletion_in_progress`.

Reply:
Got it, thank you for that context Mark! Will use `pending_deletion` then!